### PR TITLE
Run --help on all scripts during ci

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR modifies the CI configuration to run `--help` on all scripts under `bin/`, just to check that we can. This should catch some annoying errors (missing imports, syntax errors, etc).

**NOTE:** the `--help` runs do not contribute to the code coverage, and should not, because just running `--help` doesn't check that anything is correct, just that its not totally broken.

### CODE REVIEW

**Comment by cdcapano** [comment]
@duncanmmacleod I'd approve this, but since I merged #1, it needs to be rebased.

**Comment by duncanmmacleod** [comment]
thanks @cdcapano, should be rebased and ready. if you merge #2 before this one, please approve anyway and I can rebase and merge myself as soon as I land back in the UK.

**Review by cdcapano — APPROVED** [review]

**Comment by cdcapano** [comment]
@duncanmmacleod Ok, I approved this; feel free to merge once you've rebased.

**Pull request merged**
